bin/reth/src/node/mod.rs

```diff
@@ -554,6 +554,10 @@ impl<Ext: RethCliExt> NodeCommand<Ext> {
 
         self.ext.on_node_started(&components)?;
 
+        // On startup, set the finalized, safe, and unsafe head block hashes to prevent
+        // the rollup client from walking back from genesis if `enable_genesis_walkback`
+        // is false. Effectively, this will cause the rollup client to start syncing from
+        // the current tip in the DB.
```

Comment:
Can we add information about why we walk back to genesis when this is true? re: our convo in slack